app/models/export.rb

```diff
@@ -0,0 +1,26 @@
+require "csv"
+
+class Export < ActiveRecord::Base
+  def self.to_csv_for_completion_report(data)
+    CSV.generate do |csv|
+      csv << ["Zip Code", "Sign-Ups(total)", "Course Title", "Completions", ]
+      data.each do |zip_code, info|
+        # binding.pry
+        zip_code = zip_code
+        sign_ups = info[:sign_ups]
+
+        values = [zip_code, sign_ups]
+        csv.add_row values
```

Comment:
Does ```add_row``` create a new row? I've generally seen things done with ```<<``` adding an array of items to the CSV as a new line. I also noticed that ```add_row``` is called twice per iteration it seems, so maybe add_row does something different?